src/ResourceManager/ApiManagement/Commands.ApiManagement.Test/Commands.ApiManagement.Test.Netcore.csproj

```diff
@@ -3,10 +3,11 @@
   <Import Project="..\..\..\..\tools\Common.Netcore.Dependencies.Test.targets" />
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
     <AssemblyName>Microsoft.Azure.Commands.ApiManagement.Test</AssemblyName>
     <RootNamespace>Microsoft.Azure.Commands.ApiManagement.Test</RootNamespace>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <IsPackabe>false</IsPackabe>
```

Comment:
IsPackable?

Reply:
It seems that is a typo, since it is missing an 'L'. It shouldn't be needed, I believe.